src/modules/organisations/organisations.service.ts

```diff
@@ -123,4 +124,17 @@ export class OrganisationsService {
       throw new InternalServerErrorException(`An internal server error occurred: ${error.message}`);
     }
   }
+
+  async getOrganizationDetailsById(orgId: string) {
+    if (!isUUID(orgId)) {
+      throw new BadRequestException('Must Provide a valid organization Id');
+    }
```

Comment:
Please make use of the global error exception helper in declaring errors

Reply:
noted.
i have implemented this.